src/System.Console/src/System/Console.cs

```diff
@@ -365,5 +366,61 @@ public static void Write(String value)
         {
             Out.Write(value);
         }
+
+        #region ctrlc
+
+        static ConsoleCancelEventHandler cancel_event;
```

Comment:
According to the [coding guidelines](https://github.com/dotnet/corefx/wiki/Contributing#c-coding-style) this should be `s_cancelEvent`